redis/src/commands/mod.rs

```diff
@@ -2186,6 +2189,116 @@ assert_eq!(b, 5);
+    #[cfg(feature = "bloom")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "bloom")))]
+    fn bf_exists<K: ToRedisArgs, V: ToRedisArgs>(key: K, value: V) {
+            cmd("BF.EXISTS")
+        .arg(key)
+        .arg(value)
+        }
+
+    /// Add  multiple entries to a bloom filter.
+    #[cfg(feature = "bloom")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "bloom")))]
+      fn bf_madd<K: ToRedisArgs, V: ToRedisArgs>(key: K, items: V) {
+        cmd("BF.MADD").arg(key).arg(items)
+    }
+
+    /// check existence of  multiple entries to a bloom filter.
+    #[cfg(feature = "bloom")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "bloom")))]
+    fn bf_mexists<K: ToRedisArgs, V: ToRedisArgs>(key: K, items: &'a [V]) {
+        cmd("BF.MEXISTS").arg(key).arg(items)
+    }
+
+  /// Returns the cardinality of a Bloom filter - number of items that were added to a Bloom filter and detected as unique
+  /// See the [Redis documentation](https://redis.io/docs/latest/commands/bf.card/) for more information.
+    #[cfg(feature = "bloom")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "bloom")))]
+    fn bf_cardinality<K: ToRedisArgs>(key: K) {
+        cmd("BF.CARD").arg(key)
+    }
+
+    /// Creates an empty Bloom filter with a single sub-filter for the initial specified
+    /// capacity and with an upper bound error_rate.
+    #[cfg(feature = "bloom")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "bloom")))]
+    fn bf_reserve<K: ToRedisArgs, E: ToRedisArgs, C: ToRedisArgs>(key: K, err_rate: E, capacity: C) {
+        cmd("BF.RESERVE").arg(key).arg(err_rate).arg(capacity)
+    }
+
+    /// capacity and with an upper bound error_rate.
+    #[cfg(feature = "bloom")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "bloom")))]
+    fn bf_reserve_options<K: ToRedisArgs, E: ToRedisArgs, C: ToRedisArgs>(key: K, err_rate: E, capacity: C, options: ScalingOptions) {
+      cmd("BF.RESERVE").arg(key).arg(err_rate).arg(capacity).arg(options)
+    }
+
+    /// Creates a new Bloom filter if the key does not exist using the specified
+    /// error rate, capacity, and expansion, then adds all specified items to the Bloom Filter.
+    #[cfg(feature = "bloom")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "bloom")))]
+    fn bf_insert<K: ToRedisArgs,  I: ToRedisArgs>(
+        key: K,
+        items: I,
+        options: InsertOptions) {
+        cmd("BF.INSERT")
+            .arg(key)
+            .arg(options)
+            .arg("ITEMS").arg(items)
+    }
+
+    /// Returns all information about a Bloom filter.
+    #[cfg(feature = "bloom")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "bloom")))]
+    fn bf_info_all<K: ToRedisArgs>(key: K) {
+        cmd("BF.INFO").arg(key)
+    }
+
+     /// Returns specific information about a Bloom filter.
+    #[cfg(feature = "bloom")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "bloom")))]
+    fn bf_info<K: ToRedisArgs>(key: K, infotype: InfoType) {
+        cmd("BF.INFO").arg(key).arg(infotype)
+    }
+
+        #[cfg(feature = "bloom")]
```

Comment:
nit: unnecessary, you have the same attribute under the doc

Reply:
👍